content/blog/2022-07-26-schema-evolution-glue-msk/index.md

```diff
@@ -0,0 +1,861 @@
+---
+title: Schema Evolution with Glue Schema Registry and Managed Streaming for Kafka (MSK) using LocalStack
+description: Find incompatibilities early or even avoid them alltogether when developing Kafka producers or consumers! Learn how to test data schema evolution by using Managed Streaming for Kafka (MSK) with the Glue Schema Registry in LocalStack.
+lead: Find incompatibilities early or even avoid them alltogether when developing Kafka producers or consumers! Learn how to test data schema evolution by using Managed Streaming for Kafka (MSK) with the Glue Schema Registry in LocalStack.
+images: ["schema-evolution-glue-msk-poster.png"]
+contributors: ["Alexander Rashed"]
+date: 2022-07-26
+tags: ['tutorial']
+leadimage: "schema-evolution-glue-msk-poster.png"
+resources:
+- src: "schema-evolution-glue-msk-flow.svg"
+---
+
+{{< img src="schema-evolution-glue-msk-poster.png" >}}
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event store and stream-processing platform.
```

Comment:
since the main story is schema evolution, maybe we could jump right into the guts and, just for the first paragraph, assume that people know what MSK, Glue, and schema evolution means? then we would have the value proposition directly in the beginning.

then add the TL/DR, and then do a background section, "MSK and Glue Basics".

WDYT?

Reply:
it would also be great to add a short bullet list after the intro paragraph about "what you will learn in this article"